frontend/src/lib/stores/sns-unsupported-filter-by-topic.store.ts

```diff
@@ -0,0 +1,44 @@
+import type { CanisterIdString } from "@dfinity/nns";
+import { get, writable, type Readable } from "svelte/store";
+
+export type UnsupportedFilterByTopicCanistersStoreData =
+  Array<CanisterIdString>;
+
+export interface UnsupportedFilterByTopicCanistersStore
+  extends Readable<UnsupportedFilterByTopicCanistersStoreData> {
+  add: (canisterId: CanisterIdString) => void;
```

Comment:
Just a suggestion to make it more obvious that this stores the `rootCanisterIds`, not the governance canisters, that don’t support filtering.
```suggestion
  add: (rootCanisterId: CanisterIdString) => void;
```